packages/@uppy/utils/package.json

```
@@ -20,7 +20,7 @@
   "exports": {
     "./package.json": "./package.json",
     "./lib/Translator": "./lib/Translator.js",
-    "./lib/EventTracker": "./lib/EventTracker.js",
+    "./lib/EventManager": "./lib/EventManager.js",
```

Can we keep both to avoid the breaking change?
```suggestion
    "./lib/EventTracker": "./lib/EventManager.js",
    "./lib/EventManager": "./lib/EventManager.js",
```